app/controllers/api/v1/notifications_controller.rb

```diff
@@ -10,8 +10,10 @@ def index
   end
 
   def create
+    requires! :id
+    current_user = User.find(params[:id])
     current_user.notifications.create(notification_params)
-    render json: { count: current_user.unread_notifications_count }
+    render json: { count: current_user.notifications.where(unread: true).count }
```

Comment:
If i use unread_notifications_count method the number here is always 20. But the database actually have 21 unread notification. 
I don't know what's wrong with this method, maybe it have a cache. So i change it.